<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SetEventLocationDialog</class>
 <widget class="QWidget" name="SetEventLocationDialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>505</width>
    <height>594</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Set Event Location</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="3" column="3">
    <widget class="QPushButton" name="button_change_location">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Change Location</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0" rowspan="2" colspan="4">
    <widget class="QRadioButton" name="radio_change">
     <property name="text">
      <string>Exchange with another event</string>
     </property>
    </widget>
   </item>
   <item row="10" column="0">
    <widget class="QComboBox" name="combo_direction">
     <item>
      <property name="text">
       <string>Retain</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Down</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Left</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Right</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Up</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="4" column="0" colspan="4">
    <widget class="QRadioButton" name="radio_variable">
     <property name="text">
      <string>Appointment with Variables</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="3">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Event:</string>
     </property>
    </widget>
   </item>
   <item row="12" column="0" colspan="4">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="button_ok">
       <property name="text">
        <string>Ok</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="button_cancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="11" column="0" colspan="4">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>490</width>
       <height>177</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="0" colspan="4">
    <widget class="QRadioButton" name="radio_direct">
     <property name="text">
      <string>Direct Appointment</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="4">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="1" column="1">
      <widget class="SwitchVariableWidget" name="widget_var_y" native="true">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Map Y:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Map X:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="SwitchVariableWidget" name="widget_var_x" native="true">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0" colspan="3">
    <widget class="QLineEdit" name="line_direct">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="8" column="0" colspan="3">
    <widget class="QComboBox" name="combo_exchange_event"/>
   </item>
   <item row="1" column="0" colspan="3">
    <widget class="QComboBox" name="combo_event"/>
   </item>
   <item row="9" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Direction:</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SwitchVariableWidget</class>
   <extends>QWidget</extends>
   <header>dialogs/switchvariablewidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>combo_event</tabstop>
  <tabstop>radio_direct</tabstop>
  <tabstop>line_direct</tabstop>
  <tabstop>button_change_location</tabstop>
  <tabstop>radio_variable</tabstop>
  <tabstop>radio_change</tabstop>
  <tabstop>combo_exchange_event</tabstop>
  <tabstop>button_ok</tabstop>
  <tabstop>button_cancel</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>radio_direct</sender>
   <signal>toggled(bool)</signal>
   <receiver>line_direct</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>40</x>
     <y>79</y>
    </hint>
    <hint type="destinationlabel">
     <x>104</x>
     <y>105</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radio_direct</sender>
   <signal>toggled(bool)</signal>
   <receiver>button_change_location</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>69</x>
     <y>75</y>
    </hint>
    <hint type="destinationlabel">
     <x>426</x>
     <y>121</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radio_variable</sender>
   <signal>toggled(bool)</signal>
   <receiver>widget_var_x</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>216</x>
     <y>149</y>
    </hint>
    <hint type="destinationlabel">
     <x>318</x>
     <y>196</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radio_variable</sender>
   <signal>toggled(bool)</signal>
   <receiver>widget_var_y</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>197</x>
     <y>151</y>
    </hint>
    <hint type="destinationlabel">
     <x>311</x>
     <y>214</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radio_change</sender>
   <signal>toggled(bool)</signal>
   <receiver>combo_exchange_event</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>99</x>
     <y>325</y>
    </hint>
    <hint type="destinationlabel">
     <x>149</x>
     <y>360</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
